helm-charts/nxrm-ha/values.yaml

```diff
@@ -1,84 +1,284 @@
-# DO NOT MODIFY. This is produced by template.
----
+# Declare variables to be passed into your templates.
+namespaces:
+  nexusNs:
+    enabled: true
+    name: "nexusrepo"
+  cloudwatchNs:
+    enabled: false
+    name: amazon-cloudwatch
+  externaldnsNs: nexus-externaldns
```

Comment:
Should these values be kept?

Reply:
No this can go since we are not using it